cmd/kube-controller-manager/app/core.go

```diff
@@ -411,6 +411,7 @@ func startPodGCController(ctx context.Context, controllerContext ControllerConte
 }
 
 func startResourceQuotaController(ctx context.Context, controllerContext ControllerContext) (controller.Interface, bool, error) {
+	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "resourcequota"))
```

Comment:
```suggestion
	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "resourcequota-controller"))
```
to make it consistent across all of them, see this thread: https://groups.google.com/a/kubernetes.io/g/dev/c/I9l8SSM599s/m/PtzEMIS8BAAJ and especially attached spreadsheet with proposed names